dev/mcal/src/Gpio.cpp

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file Gpio.cpp
+ * @author
+ * @brief
+ * @version 0.1
+ * @date 2024-03-16
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+#include "mcal/inc/stm32f103xx.h"
+#include "utils/inc/Assert.h"
+#include "mcal/inc/Gpio.h"
+
+using namespace stm32::dev::mcal::gpio; // NOLINT[build/namespaces]
+using namespace stm32::registers::gpio; // NOLINT[build/namespaces]
+
+#define ISMODE_VALID(MODE) ((MODE == kIN_analog)  || \
```

Comment:
Indent the code like : 
#define ISMODE_VALID(MODE) ((MODE == kIN_analog)   || \
                       (MODE == kIN_floating)                               || \
                       (MODE == kIN_pullup)                                 || \
                       (MODE == kIN_pulldown)                            || \
                       (MODE == kOP_pushpull_10MHZ)               || \
                               ................................................
                       (MODE == kOP_pushpull_2MHZ)                 ||\ 
                       (MODE == kaf_opendrain_50MHZ))